desktop/main.js

```diff
@@ -282,6 +310,8 @@ const mainWindow = () => {
                     browserWindow.setTitle('New Expensify');
                 }
 
+                const PASTE_AS_PLAIN_TEXT_EDIT_MENU_ID = 'pasteAsPlainText';
```

Comment:
I don't see this being used multiple places, so we don't need to add it to constant.  Is this giving you any lint errors?

Reply:
I thought it was going to be needed more than once initially. Fixed it